src/main/asciidoc/futures.adoc

```diff
@@ -24,6 +24,13 @@ In most cases, you don't need to create promises yourself in a Vert.x applicatio
 
 However, if, in your codebase, you have legacy methods which use callbacks, you can leverage the fact that a promise extends {@link io.vertx.core.Handler io.vertx.core.Handler<io.vertx.core.AsyncResult>}:
 
+[source,$lang]
+----
+{@link examples.CoreExamples#promiseAsHandlerShort}
+----
+
+{@link io.vertx.core.Future#future} is a short form of this verbose code:
+
```

Comment:
This comment and the example below are probably no longer necessary. Can you remove them?